profiles/orcid.py

```diff
@@ -1,20 +1,52 @@
 import json
 import logging
+from urllib.parse import quote
 
-import requests
-from requests import RequestException, Response
+from requests import RequestException, Response, post, request
 
 API_VERSION = 'v2.0'
 LOGGER = logging.getLogger(__name__)
 
+PRIVATE_SCOPE = '/webhook'
+PUBLIC_SCOPE = '/read-public'
```

Comment:
These are very confusing, they're not 'public' and 'private'.

Reply:
OK, do you have a suggestion? something like:

`WEBHOOK_SCOPE`
`READ_PUBLIC_SCOPE`